### PR TITLE
Update braintree-web to 3.99.2

### PR DESCRIPTION
**Pull request by jplukarski**
### Summary

Updates braintree-web to the latest version

### Checklist

- [x] Added a changelog entry

- @jplukarski

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "@braintree/event-emitter": "0.4.1",
     "@braintree/uuid": "0.1.0",
     "@braintree/wrap-promise": "2.1.0",
-    "braintree-web": "3.99.0"
+    "braintree-web": "3.99.2"
```
Comment (SancSalix) [inline package.json:72]: Obvi 3.99.1 never happened

**Review by SancSalix — APPROVED** [review]
🌟

File: package.json (hunk 1 above)
Comment (jplukarski) [reply package.json:72]: 🤫

**Review by cgdibble — APPROVED** [review]

**Comment by ibooker** [comment]
A PR to update to `braintree-web` 3.101.3 already merged to main.

**Pull request closed without merging**